File: SCALA/Cap05/c05ex06.scala
// c05ex06.scala

import scala.io.StdIn._

object Main {
  def main(args: Array[String]): Unit = {
    var RESP: Char = 'S'
    var N, R: Int = 0

    do {
      print("Entre um valor numerico: ")
      N = readInt()

      R = N * 3

      println("Resultado = " + R + "\n")

      print("Deseja continuar? (S/N): ")
      RESP = Character.toUpperCase(readChar())

      println()
    } while (!(Character.toUpperCase(RESP) != 'S'))

    print("\nTecle <Enter> para encerrar... ")
    readLine()
  }
}

File: KOTLIN/Cap05/c05ex06.kt
// c05ex06.kt

fun main() {
    var RESP: Char
    var N: Int
    var R: Int

    do {
        print("Entre um valor numerico: ")
        N = readLine()?.toInt() ?: 0

        R = N * 3

        println("Resultado = $R\n")

        print("Deseja continuar? (S/N): ")
        RESP = readLine()?.uppercase()?.get(0) ?: 'N'

        println()
    } while (!(RESP != 'S'))

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
